### ISSUE TITLE
[TF2] "Disallow items with special qualities" does not exclude killstreak items

### ISSUE THREAD

**Issue by mock--** [ValveSoftware/Source-1-Games#4799]
When selecting items to use in crafting, even when the "disallow items with special qualities" box is checked, unique items with killstreaks still appear.

**Comment by Jgr9** [comment]
Yeah, I've had a Killstreak Machina I've had for a long time and I always fear accidentally crafting it.
